VAMobile/src/components/ActivityButton.tsx

```diff
@@ -28,7 +26,7 @@ const ActivityButton: FC<ActivityButtonProps> = ({ title, subText, deepLink }: A
     px: theme.dimensions.buttonPadding,
     backgroundColor: theme.colors.buttonBackground.activityButton as BackgroundVariant,
     style: {
-      shadowColor: colors.black,
+      shadowColor: 'black',
```

Comment:
While looking at this PR to answer Ameet's question, just noting there is a black token if you wanted to lean on it instead of having some color strings.

Reply:
Good callout, updated this to use `vadsColorBlack`